.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: "bundler"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    labels:
+      - "c: dependencies"
```

Comment:
The current label name in this repo is `dependencies` (without the `c: ` prefix):
https://github.com/heroku/heroku-buildpack-ruby/labels

Not sure if having the prefix here was intentional or not? :-)